app/database/item_store.go

```diff
@@ -69,7 +69,7 @@ func (s *ItemStore) HasManagerAccess(user *User, itemIDs ...int64) (found bool,
 	err = s.GroupItems().MatchingUserAncestors(user).
 		WithWriteLock().
 		Where("idItem IN (?) AND (bManagerAccess OR bOwnerAccess)", itemIDs).
-		PluckFirst("COUNT(DISTINCT idItem)", &count).Error()
+		Select("COUNT(DISTINCT idItem)").Count(&count).Error()
```

Comment:
What I meant with the count was something allowing `s.GroupItems().Where("idItem IN (?).Count("DISTINCT idItem")` directly. But maybe that's not a good idea.

Reply:
Gorm cannot do that. See here https://github.com/jinzhu/gorm/issues/970